static/docs/user-guide/install/completion.md

```diff
@@ -65,10 +65,6 @@ install and configure shell completion.
 
 ## Bash completion on Ubuntu
 
-On a Linux system bash-completion should work out of the box, but if you are
```

Comment:
not kinda the way to address this I expected :) Why can't we mention that it should work out of the box if it was installed from the package?

Reply:
> Why can't we mention that it should work out of the box if it was installed from the package?

If it works out of the box, these instructions still don't harm anything, because they just say "make sure that this is so and so". So, everything is still ok,

Reply:
> because they just say, make sure that this is so and so

they (almost all of them) do not, they specifically mention to do a step a step installation. It can save my time by not even reading the first step and not running the command. You went from one extreme to another for a reason I don't quite understand to be honest.

Reply:
>  for a reason I don't quite understand to be honest.

Installing from the package still does not guarantee that bash completion should work out of the box. If you install it on a VPS on the cloud this would not be enough.

Reply:
thus `_should_ be working`, not `must` - it's enough to try it before jumping into the instruction.

I still don't understand why do you use a very specific term `VPS`. I can imagine that `bash-completion` is not installed by default on a variety of Linuxes. Or am I missing something? 

Btw, @efiop, can we install (and should?) the necessary deb packages automatically to make it work? Also, will it start working automatically if I install the necessary deb packages after I installed DVC deb?

Reply:
> I still don't understand why do you use a very specific term VPS. I can imagine that bash-completion is not installed by default on a variety of Linuxes. Or am I missing something?

It **is** installed by default on almost all versions of Linux. Surprisingly it is not installed on ubuntu servers  offered on some clouds.

>  will it start working automatically if I install the necessary deb packages after I installed DVC deb?

There is also the part of enabling it on `~/.bashrc`. Installing `bash-completion` does not guarantee that your `~/.bashrc` will have the right configuration.

Reply:
@dashohoxha 

> It is installed by default on almost all versions of Linux. Surprisingly it is not installed on ubuntu servers offered on some clouds.

I suppose you are talking about bash-completion package, not dvc-specific completion. That  makes sense.

> Btw, @efiop, can we install (and should?) the necessary deb packages automatically to make it work? Also, will it start working automatically if I install the necessary deb packages after I installed DVC deb?

Yes, we could totally specify `bash-completion` package as a dependency. Probably as simple as `--deb-pre-depends bash-completion`. But I'm not sure if it is a common practice to include that as a mandatory dependency, since bash completion is optional. Need to do the research there.